geometry_msgs/Point input     # the input point
---
bool result         # whether the point is in a SOMA ROI or not
